covid-19-support/src/components/Header.vue

```diff
@@ -18,6 +18,20 @@
         <b-nav-item right @click="$bvModal.show('about-us')">
           {{ $t('about.linktext') }}
         </b-nav-item>
+        <b-nav-item right class="icons">
+          <b-link href="https://www.facebook.com/nccovidsupport">
```

Comment:
These links need to be stored in the `theme.content.js` files found in the two Theme folders so that they can be different for each site that replicates the solution.

Reply:
can you elaborate on this part please

Reply:
Yes of course, this solution is used by multiple groups to built their sites and we are trying to ensure the whole solution is easy to re-skin for a different group to use the site for their local area.

If you look in your solution at this folder:
`covid-19-support\src\themes`

You will see we have two Themes defined, one produces the NC Covid Support website:

https://nccovidsupport.org

And the other produces a demo site ( that we are slowly working on, to ensure the whole solution can easily be skinned ) so far we have made sure all the colors are in the theme and we are pulling out all the configuration and text elements.

In both of those folders you will find a file called `theme.config.js`  ( Previously I mentioned the `theme.content.js`. Now I have had a second look I think this was the wrong file to suggest, sorry! )

I would like to see a new section created at the top level ( I would put it after the `settings` element and before the `data` element. ) that looks like this:

```
socialMedia: {
  twitter: 'nccovidsupport',
  instagram: 'nccovidsupport',
  facebook: 'https://www.facebook.com/nccovidsupport'
}
```

Facebook needs to be the full URL as sometimes people don't have a nice URL name.

Take a look at the `App.vue` file and you should see how these items are pulled out as there are quite a few examples. You will need to pull them out in the same way and pass them to the Header.vue file ( if they exist ).

( Just search for `theme` in the `App.vue` file and you will find a lot of examples.

Cheers, Chris